backend/es/tests/test_alias.py

```diff
@@ -1,38 +1,30 @@
-import pytest
-
+from addcorpus.models import Corpus
 from es.es_alias import alias, get_highest_version_number
-from es.es_index import create
-from addcorpus.python_corpora.load_corpus import load_corpus_definition
 
 
-def test_alias(es_alias_client):
-    corpus_definition = load_corpus_definition('times')
-    assert corpus_definition.es_index == 'times-test'
-    alias('times', corpus_definition) # create an alias ianalyzer-times
-    res = es_alias_client.indices.get_alias(name=corpus_definition.es_index)
+def test_alias(db, es_alias_client):
+    corpus = Corpus.objects.get(name='times')
+    assert corpus.configuration.es_index == 'times-test'
+    alias(corpus)  # create an alias ianalyzer-times
+    res = es_alias_client.indices.get_alias(name=corpus.configuration.es_index)
     assert res.get('times-test-2') is not None
 
+
 def test_alias_with_clean(es_alias_client):
```

Comment:
shouldn't this test also have the `db` fixture? Then again, if it doesn't fail...